framework/common/WindowContext.kt

```diff
@@ -29,20 +29,20 @@ sealed interface WindowContext {
     class Test(override var height: Int, override var width: Int, override var x: Int, override var y: Int): WindowContext
 
     class Compose(val window: ComposeWindow): WindowContext {
-        override var height: Int
+        override var height
             get() = window.size.height
-            set(value) { value }
+            set(value) { height = value }
 
         override var width: Int
-            get() = window.size.height
-            set(value) { value }
+            get() = window.size.width
```

Comment:
Whoops. Glad we don't measure anything widthwise within Studio (though we would have caught this earlier if that were the case.)